avocado/core/task/runtime.py

```diff
@@ -212,6 +213,138 @@ def get_pre_tasks_from_runnable(cls, runnable, status_server_uri=None, job_id=No
         return pre_test_tasks
 
 
+class PostRuntimeTaskPrototype(RuntimeTask):
+    def __init__(self, task, parent_task, status_server_uri=None, job_id=None):
+        super().__init__(task)
+        self._parent_task = parent_task
+        self._status_server_uri = status_server_uri
+        self._job_id = job_id
+
+    @classmethod
+    def from_runnable(  # pylint: disable=W0221
+        cls, post_runnable, parent_task, status_server_uri=None, job_id=None
+    ):
```

Comment:
any reason for adopting this new format instead?

```python
    @classmethod
    def from_runnable(cls, post_runnable, parent_task,
                                      status_server_uri=None, job_id=None):
```

I'm not sure if this is a new convention or something...

Reply:
The fact we are disabling W0221 here give-me some "orange flags". Probably you are not introducing this here, and just following the same from "PreRuntimeTask". But the fact this method signature is different from the RuntimeTask is strange to me.